packages/tar-format/tar-format.0.5.0/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "1.2"
+maintainer:   "dave@recoil.org"
+authors:      [ "Dave Scott" "Thomas Gazagnaire" "David Allsopp" ]
+tags:         ["org:xapi-project" "org:mirage"]
+homepage:     "https://github.com/mirage/ocaml-tar"
+bug-reports:  "https://github.com/mirage/ocaml-tar/issues"
+dev-repo:     "https://github.com/mirage/ocaml-tar.git"
+
+build: [
+  ["ocaml" "setup.ml" "-configure"
+     "--prefix" prefix
+     "--%{lwt:enable}%-lwtunix"
+     "--%{mirage-types-lwt:enable}%-mirage" ]
+  ["ocaml" "setup.ml" "-build"]
+]
+build-test: [
+  ["ocaml" "setup.ml" "-configure"
+    "--enable-tests" "--enable-lwtunix" "--enable-mirage"]
+  [make "test"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+remove:  ["ocamlfind" "remove" "tar"]
+
+depends: [
+  "ocamlfind"
```

Comment:
Could be `{build}`? Also, you probably rely on `ocamlbuild` if you are using OASIS.